target/linux/ramips/dts/mt7621_hanyang_hyc-g920.dts

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "hanyang,hyc-g920", "mediatek,mt7621-soc";
+	model = "CJ-Hello HYC-G920";
```

Comment:
We prefer the model string to match `"$DEVICE_VENDOR $DEVICE_MODEL"` from its device recipe if possible.

Reply:
these are Device photos

https://ibb.co/r3gGB0r
https://ibb.co/wzhzv6W
https://ibb.co/XjfmwjY

Reply:
I mean, please use "Hanyang HYC-G920".